save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/SettingsView.kt

```diff
@@ -66,14 +67,22 @@ typealias FieldsStateSetter = StateSetter<SettingsInputFields>
 @Suppress("MISSING_KDOC_CLASS_ELEMENTS")
 external interface SettingsProps : PropsWithChildren {
     /**
-     * Currently logged in user or null
+     * Currently logged-in user or null
      */
     var userInfo: UserInfo?
 
     /**
      * just a flag for a factory
      */
     var type: FrontendRoutes
+
+    /**
+     * After updating user information we will update userSettings without re-rendering the page
+     * PLEASE NOTE: THIS PROPERTY AFFECTS RENDERING OF WHOLE APP.KT
```

Comment:
Does it? I have heart that react re-renders only components that have changes (their props or/and states).

Reply:
yes, I mean that it will affect all components, because we put props.userInfo mostly everywhere

Reply:
Got it